pkg/packetfilter/configure/configure.go

```diff
@@ -42,3 +42,7 @@ func DriverFromGlobalConfig() {
 		packetfilter.SetNewDriverFn(iptables.New)
 	}
 }
+
+func IPtablesCleanupIsNeeded() bool {
```

Comment:
We  can make this more generic, ideally to query what driver is being used. It would be nice if we could check `packetfilter.GetNewDriverFn() == nftables.New` but Go doesn't allow that. Instead, we can move the [DriverType](https://github.com/submariner-io/submariner/blob/00184da5be3fa0df959292c5208f53bb3e3a9b3e/pkg/packetfilter/configure/configure_suite_test.go#L36) constants to  this file and add a `GetDriverType` function (that's also used by `DriverFromGlobalConfig`).

Then, to check is IP tables cleanup is needed, callers can check

```
if pfconfigure.GetDriverType() == pfconfigure.IPTables {
    return
}

// do cleanup
```